services/explorer/src/pages/PBFTData/PBFTDataContainer.tsx

```diff
@@ -70,12 +70,12 @@ const PBFTDataContainer = (): JSX.Element => {
               component='h6'
               style={{ fontWeight: 'bold', wordBreak: 'break-all' }}
             >
-              {identifier}
+              {zeroX(blockData.hash)}
```

Comment:
Double-check if we need to add the 0x in front of the hash. I don't think we do.
Also, we need to make sure the blockData is here otherwise it will crash. So add blockData?.hash. And do this in all DataRow elements where it is possible.
I had to do the same thing in TransactionData.tsx because it crashed.